common/src/components/WDGInput.vue

```diff
@@ -75,6 +75,11 @@ export default {
 		onInputLocalEvent () {
 			this.$emit('update:valueReturn', this.valueReturn)
 		}
+	},
+	computed: {
+		showRequiredStar () {
+			return this.isRequired && !!this.$slots.label
```

Comment:
deux points d'exclamation ?

Reply:
oui, je ne sais pas pourquoi mais c'est comme ça u'on sait si un slot est vide ou pas, avec un seul ! ça ne marche pas. J'ai trouvé ça là : https://stackoverflow.com/questions/44077277/only-show-slot-if-it-has-content

Reply:
Chelou !